tests/functional/test_unattended_access.py

```diff
@@ -37,6 +34,21 @@ def api_key():
     return key
 
 
+def get_patient_request(headers: dict):
+    return requests.get(
+        (
+            "https://internal-dev.api.service.nhs.uk/"
+            "personal-demographics/"
+            "Patient?"
+            "family=Smith"
```

Comment:
Requests provides the `params` keyword arg so you can pass a dict instead of writing it out in the URL